src/material-experimental/mdc-helpers/_focus-indicators.scss

```diff
@@ -58,6 +58,15 @@
     margin: 5px;
   }
 
+  // These components have to set `border-radius: 50%` in order to support density scaling
+  // which will clip a square focus indicator so we have to turn it into a circle.
+  .mat-mdc-radio-button,
+  .mat-mdc-checkbox {
+    .mat-mdc-focus-indicator::before {
```

Comment:
Developers often place components inside checkboxes & radio buttons (even though they arguably shouldn't for accessibility reasons). These styles as-is will cause all nested MDC focus indicators within radio-button & checkbox to be circular (which I don't think we want). Can we write this selector to specifically target the focus indicators owned by these controls?

Reply:
Good point 👍 Changing...